code/_helpers/global_lists.dm

```diff
@@ -117,6 +118,12 @@ var/global/list/cloaking_devices = list()
 			else
 				facial_hair_styles_male_list += H.name
 				facial_hair_styles_female_list += H.name
+				
+	//Body markings - Initialise all /datum/sprite_accessory/marking into an list indexed by marking name
+	paths = typesof(/datum/sprite_accessory/marking) - /datum/sprite_accessory/marking

```

Comment:
Should clarify that `subtypesof(/datum/sprite_accessory/marking)` is the same as `typesof(/datum/sprite_accessory/marking) - /datum/sprite_accessory/marking`

And would replace the whole thing.